autopush/db.py

```diff
@@ -125,7 +125,7 @@ def make_rotating_tablename(prefix, delta=0, date=None):
     month delta."""
     if not date:
         date = get_month(delta=delta)
-    return "{}_{}_{}".format(prefix, date.year, date.month)
+    return "{}_{:04d}_{:02d}".format(prefix, date.year, date.month)
```

Comment:
@bbangert Ooh, you're right. Remind me to fix it in 7,984 years. 

(I'll go back to slowly drawing red circles around Tuesday, 19 January 2038)